docs/release.md

```diff
@@ -80,6 +82,16 @@ The last step of the release process creates artifacts for the new version of th
 2. `commitChangesToNewBranch failed: invalid merge` -- This is a [known issue](https://github.com/open-telemetry/opentelemetry-go-build-tools/issues/47) with our release tooling. The current workaround is to clone a fresh copy of the repository and try again. Note that you may need to set up a `fork` remote pointing to your own fork for the release tooling to work properly.
 3. `could not run Go Mod Tidy: go mod tidy failed` when running `multimod` -- This is a [known issue](https://github.com/open-telemetry/opentelemetry-go-build-tools/issues/46) with our release tooling. The current workaround is to run `make gotidy` manually after the multimod tool fails and commit the result.
 4. `Incorrect version "X" of "go.opentelemetry.io/collector/component" is included in "X"` in CI after `make update-otel` -- It could be because the make target was run too soon after updating Core and the goproxy hasn't updated yet.  Try running `export GOPROXY=direct` and then `make update-otel`.
+5. `error: failed to push some refs to 'https://github.com/open-telemetry/opentelemetry-collector-contrib.git'` during `make push-tags` -- If you encounter this error the `make push-tags` target will terminate without pushing all the tags. Using the output of the `make push-tags` target, save all the un-pushed the tags in `tags.txt` and then use this make target to complete the push:
+
+   ```bash
+   .PHONY: temp-push-tags
+   temp-push-tags:
+       for tag in `cat tags.txt`; do \
```

Comment:
```suggestion
       for tag in `git tag | grep vX.Y.Z`; do \
```

This is the approach I usually take for Go SDK releases.  Push all tags of the appropriate version.  Hopefully easier than constructing a list of tags by hand.

Reply:
Luckily I could copy and paste the list into the file bc the target printed it out, but I like this idea more.